ci-operator/step-registry/aro/ocm/register/aro-ocm-register-commands.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+PULL_SECRET_FILE=${PULL_SECRET_FILE:="${CLUSTER_PROFILE_DIR}/pull-secret"}
```

Comment:
Is this file of pull-secret existing? What account it belongs to?

Reply:
yes the pull secret file already exists in the Prow job and it is passed to the cluster at the time the cluster is provisioned. There is a filter in the ARO RP which removes the cloud.openshift.com secret, as of this moment the script simply puts it back after the cluster is up and running.